src/lib/GameforgeAPI.ts

```diff
@@ -31,26 +32,35 @@ export class GfAccount {
         locale,
         gfLang,
         installationId,
-        identity,
+        fingerprint,
         certStore,
     }: {
         locale: GfLocale;
         gfLang: GfLang;
         installationId: string;
-        identity: Identity;
+        fingerprint: Fingerprint;
         certStore: CertificateStore;
     }) {
         this.locale = locale;
         this.gfLang = gfLang;
         this.installationId = installationId;
 
-        this.identity = identity;
+        this.fingerprint = fingerprint;
         this.certStore = certStore;
 
         this.gameSessionId = uuid();
     }
 
     async authenthicate(email: string, password: string): Promise<boolean> {
+        if (this.clientVersion == null) {
+            this.clientVersion = (await getGameforgeClientVersion()) as GameforgeClientVersion;
+        }
```

Comment:
This can be simplified to:

```suggestion
	this.clientVersion = this.clientVersion ?? (await getGameforgeClientVersion())!;

```

First, we remove the if-statement by using [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) and then we remove the type cast by using [non-null assertion operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator).